includes/gateways/paypal/admin/connect.php

```diff
@@ -37,6 +37,20 @@ function connect_settings_field() {
 		/**
 		 * Show Connect
 		 */
+
+		/*
+		 * If we have Partner details but no REST credentials then that most likely means
+		 * PayPal wasn't opened in the modal. We'll show an error message about popups.
+		 */
+		if ( get_partner_details() ) {
+			?>
+			<div class="notice notice-error inline">
+				<p>
+					<?php esc_html_e( 'Connection failure. Please ensure browser popups are enabled then click the button below to connect again. If you continue to experience this error, please contact support.', 'easy-digital-downloads' ); ?>
```

Comment:
```suggestion
					<?php wp_kses( printf( __( '%sConnection failure:%s Please ensure browser popups are enabled, then click the button below to connect again. If you continue to experience this error, please contact support.', 'easy-digital-downloads' ), '<strong>', '</strong>' ), array( 'strong' => array() ) ); ?>
```
That's sloppy and doesn't include translator comments, but what would you think of adjusting to match the general notices pattern (strong: notice)?

Note: if my site is connected in test mode, and I enable Live mode, I do get this error. My only concern about that process is that you only see this notice on the PayPal settings page. If I switch from Test to Live without visiting that screen, I won't know there is an issue. I think we wrapped an inline notice with the Test Mode checkbox in Stripe to account for this kind of scenario--wonder if we should here as well?

Reply:
Updated the message here: https://github.com/easydigitaldownloads/easy-digital-downloads/pull/8812/commits/8e584adf9396cdf617ebf89a97ee58ce287e4369

Will address your other question separately.